### PR TITLE
My personal portfolio mobile version.

### PR DESCRIPTION
**Pull request by AdrianScharff**
• Added HTML codes
• Added CSS Codes
• Used flex command

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ <header> <a href="">My logo</a>
+    <nav> 
+        <a href="#" class="fa fa-bars"></a>
+        </nav>
+        <img class="topim" src="./images/" alt=""></header>
+        
+    <section>
+        <h1>I'm Adrian. <br> Glad to see you </h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <a href="#">LET'S CONNECT</a>
```
Comment (mahbubul14) [inline index.html]: - [ ] You have a horizontal scrollbar on your web app. Kindly fix the issue. :+1: 

https://github.com/AdrianScharff/My-portfolio-setup/assets/70483835/12dbbe02-5908-42ad-97a0-501d955f0d68

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        
+        <span class="icons">
+            <img src="./images/Vector.png" alt="">
+            <img src="./images/Linkedin icon.png" alt="">
+            <img src="./images/Vector (3).png" alt="">
+            <img src="./images/Vector (4).png" alt="">
+            <img src="./images/Vector (5).png" alt="">
+        </span>
```
Comment (mahbubul14) [inline index.html]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly work on it. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
**Hi @AdrianScharff ,**

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ]  You have used the correct template for the readme but it is not updated according to the project. Please update your README file like getting started, tech stack, Author, License section. etc.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by AbdusSattar-70 — COMMENTED** [review]
### INVALID 
Hi @AdrianScharf , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that did not pass all the linter checks. Please proceed to fix the linter issues before resubmitting the code for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  top: 50%;
+  bottom: 50%;
+
+  /* Base / 300 - Tertiary
+For small, less significant text and placeholders.
+*/
+
+  /* border: 2px solid #ACB7C3; */
+}
+
```
Comment (Bantealem) [inline style.css]: [OPTIONAL] Please remove commented code from your code. Leaving commented code in your code base is not the best practice. Kindly, remove it

File: style.css [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  height: 604px;
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+section h1 {
+  /* Primary Text */
```
Comment (Bantealem) [inline style.css]: [OPTIONAL] Please remove commented code from your code. Leaving commented code in your code base is not the best practice. Kindly, remove it

File: style.css [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+  width: 327px;
+  height: 150px;
+
+  /* Header / 2 */
```
Comment (Bantealem) [inline style.css]: [OPTIONAL] Please remove commented code from your code. Leaving commented code in your code base is not the best practice. Kindly, remove it

File: style.css [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding-left: 5%;
+
+  /* or 130% */
```
Comment (Bantealem) [inline style.css]: [OPTIONAL] Please remove commented code from your code. Leaving commented code in your code base is not the best practice. Kindly, remove it

File: style.css [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+  margin-top: 5%;
+  list-style: none;
+}
+
+.icons img {
+  width: 20px;
+  height: 20px;
+}
+
+/* this is the second portfolio */
```
Comment (Bantealem) [inline style.css]: [OPTIONAL] Please remove commented code from your code. Leaving commented code in your code base is not the best practice. Kindly, remove it

**Review by Bantealem — APPROVED** [review]
# Status: Approved 
Hi @AdrianScharff , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] No Linter error
- [x] Professional decumenation
- [x] Descriptive PR title and summary 
## Optional suggestions
- [ ] Although the previous reviewer didn't ask you to implement the responsiveness of the project, I strongly recommend you make it responsive. You can use the percentage for `width`.
- [ ] The link for the social media should redirect the user to your corresponding social media account. Kindly signup for each account and replace the current link with your account.
- [ ] Kindly check the inline optional comments
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**